homeassistant/components/ituran/strings.json

```diff
@@ -0,0 +1,45 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "id_or_passport": "ID or passport number",
+          "phone_number": "Mobile phone number",
+          "mobile_id": "Mobile ID (leave empty if unknown)"
+        },
+        "data_description": {
+          "id_or_passport": "The goverment ID or passport number provided when registering with Ituran.",
+          "phone_number": "The mobile phone number provided when registering with Ituran. A one-time password will be sent to this mobile number.",
+          "mobile_id": "A unique ID used to identify the Ituran client. This can be shared between multiple clients to allow them all access to the Ituran services simultaniously. If one is not provided, it will be generated automatically."
+        }
+      },
+      "otp": {
+        "data": {
+          "otp": "OTP"
+        },
+        "data_description": {
+          "otp": "A one-time-password sent as a text message to the mobile phone number provided before."
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "invalid_otp": "OTP invalid",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
+      "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]",
+      "id_mismatch": "ID or passport number are not allowed to change"
```

Comment:
If it's not allowed to change we should not ask the user for it again. If you have to refactor the reauth flow for this I suggest we keep the reauth flow out for now and add it in a follow up PR

Reply:
I actually copied this from another integration. If I remove the reauth flow completely, what would happen after a month when the token is invalidated and the coordinator will raise a `ConfigEntryAuthFailed` exception? Don't I need a re-auth flow in this case?

I think I can, relatively easily, pre-populate the fields with the existing configuration for the user. Would that be good enough for now and, in a future PR, remove the ID field from the form?

Reply:
You can raise a ConfigEntryError for now so it doesn't retry. I'm interested what the other integration is though

Reply:
BMW Connected Drive. Seemed close to what I wanted so I took some inspiration from it...

So, would you prefer I remove the reauth completely, pre-populate the configuration (I think that's what the BMW integration does) or something else?

Reply:
remove it for now so this gets merged and we'll discuss specifically in a follow up PR

Reply:
Let's remove it for now. I'll check with the BMW codeowners, thanks. I missed that when doing the quality scale review

Reply:
Now you've made me a snitch... :)

Anyway, removed the reauthentication flow.
Just so I know for next one, what would be expected in this case, the user would click on the reauthentication button, should I immediately send the request for an OTP code and just show the form to enter it or should there be some form saying that the flow will send a new OTP request with OK/cancel buttons and only after the user approves, do I send the request and show the OTP entry form?